src/hooks/useStyles.tsx

```diff
@@ -0,0 +1,90 @@
+import { createStyles, makeStyles } from '@material-ui/core/styles';
```

Comment:
duplicate file

Reply:
yeah, having some issues with git not picking up changes in casing for directory names. will fix

Reply:
okay got it that was a nightmare